apps/dashboard/src/components/subscribers/schema.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from 'zod';
+import { isValidPhoneNumber } from 'react-phone-number-input';
+
+export const SubscriberFormSchema = z.object({
+  firstName: z.string().optional(),
+  lastName: z.string().optional(),
+  email: z.string().email().optional().nullable(),
```

Comment:
we shouldn't allow empty strings

Reply:
ok, good point, but let's do truncation on the sides, now I can type `"      "`

Reply:
Will trim it on form save 👌🏼